common.php

```diff
@@ -48,10 +48,33 @@ function crash($errorCode, $data = null) {
 	define("SUMMER", 1);
 	define("FALL", 2);
 
+	$season_name = [
+		0 => "Spring",
+		1 => "Summer",
+		2 => "Fall"
+	];
+	
+	$season_num = [
```

Comment:
Looks like $season_num isn't used anywhere. Also we shouldn't need both it and $season_name.